src/Rebugger.jl

```diff
@@ -17,15 +17,8 @@ include("debug.jl")
 include("ui.jl")
 include("deepcopy.jl")
 
-# Set up keys that enter rebug mode from the regular Julia REPL
-# This should be called from your ~/.julia/config/startup.jl file
-function repl_init(repl)
-    repl.interface = REPL.setup_interface(repl; extra_repl_keymap = rebugger_modeswitch)
-end
-
```

Comment:
Removing this made the logic in `add_keybinding` and `startup.jl` a bit easier. 

Was I correct in assuming that `repl_init()` wasn't really needed, since we can set the keybindings we need with `LineEdit.add_nested_key!` anyway?

Reply:
>Perhaps a more serious concern is that if you don't set them the way you want them initially, then the REPL is "polluted" with the choices that were made before the customization. Thus user's aren't going to be able to avoid a problematic choice, only turn on a new alternative.

Yeah -- I tried to find a way to remove an old keybinding whenever a new one is added, but it turned out to be a hacky solution: `REPL.LineEdit` doesn't seem to have a function à la `remove_nested_key!`, so I ended up experimenting with manually removing the keybinding from the nested Dict data structure. That seemed to rely too much on the internal REPL data structures, though, so I abandoned it -- after all, `add_nested_keys!` is able to override existing keybindings, which should mitigate the pollution issue.

Reply:
Another approach would be to use `add_nested_keys!` to assign a noop to the old keybinding. But that seems less than ideal, since it makes it look like a keybinding is not taken, when in fact it is.

Reply:
I think the best is to try to set it up so that the user can customize the choices before initializing the REPL; then there's nothing that needs to be deleted.  For people who launch it after the REPL is started, yes, there will be some redundancy. I think we just have to live with that.

Reply:
Sounds good!